guides/common/modules/proc_using-freeipa-credentials-to-log-in-to-the-project-hammer-cli.adoc

```diff
@@ -1,60 +1,54 @@
 [id="Using_{FreeIPA-context}_credentials_to_log_in_to_the_{project-context}_Hammer_CLI_{context}"]
-= Using {FreeIPA} credentials to log in to the {Project} Hammer CLI
+= Logging in to Hammer CLI with {FreeIPA} credentials
 
-This section describes how to log in to your {Project} Hammer CLI with your {FreeIPA} (IdM) login and password.
+Authenticate to the {Project} Hammer CLI with your {FreeIPA} username and password.
 
 .Prerequisites
-* You have enrolled your {ProjectServer} into {FreeIPA} and configured it to use {FreeIPA} for authentication.
-More specifically, you have enabled access both to the {ProjectWebUI} and the {Project} API.
+* You have configured Hammer CLI to accept {FreeIPA} credentials.
+// See xref:configuring-hammer-cli-to-accept-{FreeIPA-context}-credentials_{context}[].
 ifndef::orcharhino[]
-For more information, see {InstallingServerDocURL}Using_FreeIPA_{project-context}[Using {FreeIPA}] in _{InstallingServerDocTitle}_.
+For more information, see {InstallingServerDocURL}configuring-hammer-cli-to-accept-{FreeIPA-context}-credentials_{context}[Configuring Hammer CLI to accept {FreeIPA} credentials] in _{InstallingServerDocTitle}_.
```

Comment:
Here I get 404 but I guess it's not published yet?

Reply:
Might be, but I also changed a few of the anchors. I'll make sure to check the links next week.